src/Agent.Worker/WorkerCommandManager.cs

```diff
@@ -52,6 +53,14 @@ public bool TryProcessCommand(IExecutionContext context, string input)
             IWorkerCommandExtension extension;
             if (_commandExtensions.TryGetValue(command.Area, out extension))
             {
+                if (!extension.SupportedHostType.IsNullOrEmpty() &&
```

Comment:
there is no reason add a separate string that point to the same Doc, just update the doc, customer will figure out why command not found. :)